maquetacion/scss/_tech.scss

```diff
@@ -2,8 +2,10 @@
 @import 'colors';
 
 .content-cards {
+  display: flex;
+  flex-wrap: wrap;
   margin-bottom: #{$text-medium * 2};
-  padding: $text-medium #{$text-medium * 2};
+  padding: $text-medium #{$text-medium * 2} $text-medium #{$text-medium * 4};
```

Comment:
You dont need calc all of this values. You can just set a fixed padding. The same for all the values if they are not dinamic.